### DISCUSSION TITLE
Webp image inside SVG not found

### DISCUSSION THREAD

**Discussion by lamka02sk** [libvips/libvips#3624] [General]
I have an SVG file with <image> tag pointing to webp image in the same directory:
```
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.dev/svgjs" width="122" height="122" id="editor-board" viewBox="0 0 122 122" handler-size="0.305" default-text="">
  <g id="draw-area"><image width="81.4" height="122" xlink:href="chad-madden-je-ziwkxf-i-unsplash.webp" x="20.299999999999997" y="0" id="element_fRpW6L4-d5RfZ01Cu1xkO" transform="matrix(1,0,0,1,0,0)" preserveAspectRatio="none" aspect-ratio="true" class="" emoji="perfect" emoji-size="0.305"></image></g>
</svg>
```
From this SVG file, I am generating webp thumbnail:
```
vips thumbnail thumbnail.svg thumbnail.webp[Q=60,strip,preset=photo] 1000
```
The problem is, this results in an empty image. It looks like vips won't include webp image referenced in SVG. It works fine if the referenced image is for example JPG:
```
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.dev/svgjs" width="122" height="122" id="editor-board" viewBox="0 0 122 122" handler-size="0.305" default-text="">
  <g id="draw-area"><image width="81.4" height="122" xlink:href="chad-madden-je-ziwkxf-i-unsplash.jpg" x="20.299999999999997" y="0" id="element_fRpW6L4-d5RfZ01Cu1xkO" transform="matrix(1,0,0,1,0,0)" preserveAspectRatio="none" aspect-ratio="true" class="" emoji="perfect" emoji-size="0.305"></image></g>
</svg>
```
Do you have any idea why this might be happening? Am I missing any libraries? 
Thanks

**Comment by jcupitt** [comment]
Hi @lamka02sk, that's all done by librsvg, so you'd need to look into why that does not support webp hrefs.

**Reply by lamka02sk** [reply]
Thank you. I finally found the problem. The library that is responsible for loading images from hrefs - GdkPixbuff doesn't support webp images and it looks like they don't plan to include this feature anytime soon. 
To add webp support, I had to manually install https://github.com/aruiz/webp-pixbuf-loader.
